.env.example

```diff
@@ -0,0 +1,13 @@
+NODE_ENV=development
```

Comment:
What is the reason we have an example env at the root. Since we have examples per folder that's good to move there where it matches